tests/registration/proof-of-work-manager.spec.ts

```diff
@@ -46,8 +46,8 @@ describe('ProofOfWorkManager', function () {
       }
     };
 
-    const challengeNonceRefreshSpy = sinon.stub(proofOfWorkManager, 'refreshChallengeNonce').callsFake(stub);
-    const maximumAllowedHashValueRefreshSpy = sinon.stub(proofOfWorkManager, 'refreshMaximumAllowedHashValue').callsFake(stub);
+    const challengeNonceRefreshSpy = sinon.stub(proofOfWorkManager as any, 'refreshChallengeNonce').callsFake(stub);
```

Comment:
For my understanding: do you know what changed to make the type no longer inferable?

Reply:
It was either upgrading the sinon library in general or adding the `@types/sinon`. I upgraded the version because I couldn't really tell which `@types` version I should have used, so figured I'd update to latest of both.

I know on our other projects w/ the typed sinon private methods also need to be typed as any (or something that it understands).